packages/@ember/-internals/glimmer/lib/component.ts

```diff
@@ -1061,71 +1060,4 @@ Component.reopenClass({
 
 setInternalComponentManager(CURLY_COMPONENT_MANAGER, Component);
 
-if (EMBER_MODERNIZED_BUILT_IN_COMPONENTS) {
-  Object.defineProperty(Component, '_wasReopened', {
-    configurable: true,
-    enumerable: false,
-    writable: true,
-    value: false,
-  });
-
-  Object.defineProperty(Component, 'reopen', {
-    configurable: true,
-    enumerable: false,
-    writable: true,
-    value: function reopen(this: typeof Component, ...args: unknown[]): unknown {
-      if (this === Component) {
-        deprecate(
-          'Reopening the Ember.Component super class itself is deprecated. ' +
-            'Consider alternatives such as installing event listeners on ' +
-            'the document or add the customizations to specific subclasses.',
-          false,
-          {
-            id: 'ember.component.reopen',
-            for: 'ember-source',
-            since: {
-              enabled: '4.0.0',
-            },
-            url: 'https://deprecations.emberjs.com/v4.x#toc_ember-component-reopen',
-            until: '5.0.0',
```

Comment:
Per https://github.com/emberjs/ember.js/pull/19744#issuecomment-954956205 this deprecation (and the other in this file) were aimed at 4.x through the 3.x series, and that should not have been changed. A path should likely land *before* this PR lands fixing that in master and any releases impacted by the linked change.

Reply:
Done in https://github.com/emberjs/ember.js/pull/19823 and https://github.com/emberjs/ember.js/pull/19828